airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/json/JsonToAirbyteValue.kt

```diff
@@ -103,18 +107,13 @@ class JsonToAirbyteValue {
         if (!json.isObject) {
             throw IllegalArgumentException("Could not convert $json to Object")
         }
+        val objectProperties = LinkedHashMap<String, AirbyteValue>()
```

Comment:
what we discussed earlier in the week - pass through all object properties instead of filtering